CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# Change Log
+
+## [v0.8.0](https://github.com/ksonnet/ksonnet/tree/v0.8.0) (2017-12-20)
+[Full Changelog](https://github.com/ksonnet/ksonnet/compare/v0.7.0...v0.8.0)
+
+**Implemented enhancements:**
+
+- Package list/install is awkward [\#195](https://github.com/ksonnet/ksonnet/issues/195)
+- Rework demos/examples in light of \#169 [\#194](https://github.com/ksonnet/ksonnet/issues/194)
+- Consider returning the URI on `apply` for services [\#4](https://github.com/ksonnet/ksonnet/issues/4)
```

Comment:
This was closed as a "won't implement" -- I should have been more clear on the issue. Probably best to remove this item from the list.

Reply:
cool.. will do.